src/main/java/entity/Message.java

```diff
@@ -111,8 +111,12 @@ public String getTextContent() {
         return textContent;
     }
 
-    public Optional<Long> getAttachmentId() {
-        return Optional.ofNullable(attachmentId);
+    public OptionalLong getAttachmentId() {
+        if (attachmentId == null) {
+            return OptionalLong.empty();
+        } else {
+            return OptionalLong.of(attachmentId);
```

Comment:
Since the if-block returns, you can omit the `else` keyword.

Reply:
Done